styles.css

```diff
@@ -87,29 +106,63 @@ img {
   flex-direction: column;
   justify-content: space-around;
   align-items: center;
+  padding: 0;
+  margin: 0;
+
 }
 
-.search-bar-input {
+
+.search-area {
   display: flex;
-  flex-direction: row;
+  width: 40%;
 }
 
+.search-icon {
+  width: 75%;
+  height: 75%;
+  margin: 0;
+  padding: 0;
+}
+
+
 .search-button {
-  width: 5%;
-  height: 5%;
+  width: 20px;
+  height: 20px;
+  padding: 0px;
+  margin: 0;
+  box-shadow: none;
+  border-radius: 0;
+  border: 0;
+}
+
+#search-bar {
+  padding: 0px;
+  display: inline;
+  margin: 0;
+  width: 100%;
+  border: 1px solid #353567;
+  border-radius: 2px;
 }
 
+
+
 .idea-card{
   border-style: solid;
   display: grid;
   height: 15%;
   width: 15%;
 }
 
+
+/* Grid layout for idea cards. 
```

Comment:
Awesome! I def hope we can successfully merge this with my card layout. I worked for a while on the grid but was not successful.